service/discovery/azure/storage.go

```diff
@@ -502,7 +635,13 @@ func (d *azureStorageDiscovery) discoverObjectStorages(account *armstorage.Accou
 		}
 
 		for _, value := range pageResponse.Value {
-			objectStorages, err := d.handleObjectStorage(account, value)
+			// Discover diagnostic settings for the given container
+			_, err := d.discoverDiagnosticCategory(util.Deref(account.ID))
```

Comment:
If we don't use the first return parameter, do we actually need to define it?

Reply:
This is a leftover from my tests.  :+1: